draft-ietf-mls-protocol.md

```diff
@@ -2021,17 +2022,22 @@ message at the same time, by taking the following steps:
   applied at the leftmost unoccupied leaf, or appended to the right edge of the
   tree if all leaves are occupied.
 
-* Create a DirectPath using the new tree (which includes any new members).  The
-  GroupContext for this operation uses the `group_id`, `epoch`, `tree_hash`, and
-  `confirmed_transcript_hash` values in the initial GroupContext object.
+* If populating the `path` field: Create a DirectPath using the new tree (which
+  includes any new members).  The GroupContext for this operation uses the
+  `group_id`, `epoch`, `tree_hash`, and `confirmed_transcript_hash` values in
+  the initial GroupContext object.
 
-   * Assign this DirectPath to the `path` fields in the Commit.
+   * Assign this DirectPath to the `path` field in the Commit.
 
    * Apply the DirectPath to the tree, as described in
      {{synchronizing-views-of-the-tree}}. Define `commit_secret` as the value
      `path_secret[n+1]` derived from the `path_secret[n]` value assigned to
      the root node.
 
+* If not populating the `path` field: Set the `path` field in the Commit to the
+  null optional.  Define `commit_secret` as the all-zero vector of the same
+  length as a `path_secret` value would be.
```

Comment:
Why not set `commit_secret` to zero length vector? Seems easier to me.

Reply:
That would also work for me.  I just copied what TLS does, e.g., for missing PSK inputs.